src/signed_data.rs

```diff
@@ -296,6 +296,26 @@ pub static RSA_PKCS1_3072_8192_SHA384: SignatureAlgorithm = SignatureAlgorithm {
     verification_alg: &signature::RSA_PKCS1_3072_8192_SHA384,
 };
 
+/// RSA PSS signatures using SHA-256 for keys of 2048-8192 bits.
+pub static RSA_PSS_2048_8192_SHA256: SignatureAlgorithm = SignatureAlgorithm {
```

Comment:
I think we should name all of these new things with a _LEGACY_KEY or _USING_RSA_ENCRYPTION_KEY suffix or similar. Then later we can add similar algorithms using the un-suffixed names to allow people to choose to support RSA-PSS using only keys that use the RSA-PSS algorithm identifier.

Please make it clear in the documentation comment that these expect the key to be `rsaEncryption` public keys and reference https://tools.ietf.org/html/rfc3447#appendix-C.

(We don't need to add support for the RSA-PSS algorithm identifier now.)